src/DetailsView/components/overview-content/overview-help-links.tsx

```diff
@@ -2,13 +2,14 @@
 // Licensed under the MIT License.
 import * as React from 'react';
 
-import { ExternalLink } from '../../../common/components/external-link';
+import { ExternalLink, ExternalLinkDeps } from '../../../common/components/external-link';
 import { NamedSFC } from '../../../common/react/named-sfc';
 import { HyperlinkDefinition } from '../../../views/content/content-page';
-import { HelpLinkDeps } from './overview-help-section';
+
+export type HelpLinksDeps = ExternalLinkDeps;
 
 export interface HelpLinksProps {
-    deps: HelpLinkDeps;
+    deps: HelpLinksDeps;
     linkInformation: HyperlinkDefinition[];
 }
 
```

Comment:
Should we change the name of the file to be help-links to match the primary thing contained in the file (there's no overview here)?

Reply:
Same thoughts here as with the comment on `overview-help-section` deps and props types.